defaults.sh

```diff
@@ -75,7 +82,7 @@ DEF_RELCYCLE="Beta"
 DEF_VARIANT="SailfishOS-SDK"
 
 # The default version description
-# Appears in braces after Qt Creator version in the Qt Creator About dialog
+# Appears in braces after the SDK release name in the Qt Creator and QmlLive About dialog
```

Comment:
We haven't been mentioning SDK release name in the about dialogs for quite long time :) There was the issue that the compiled-in SDK release name tended to be behind the current SDK release, especially for Qt QmlLive.